py-polars/polars/series/series.py

```diff
@@ -3635,9 +3635,18 @@ def to_numpy(
         use_pyarrow: bool = True,
     ) -> np.ndarray[Any, Any]:
         """
-        Convert this Series to numpy. This operation clones data but is completely safe.
+        Convert this Series to numpy.
 
-        If you want a zero-copy view and know what you are doing, use `.view()`.
+        This operation may clone data but is completely safe.
+        Note that:
+        - data which is purely numeric AND without null values is not cloned;
```

Comment:
I think there must be a newline here for it to render properly, but I'm not 100% sure.

```suggestion
        Note that:

        - data which is purely numeric AND without null values is not cloned;
```

Reply:
you're right, thanks!

I've updated, and now it renders properly:

![image](https://github.com/pola-rs/polars/assets/33491632/26e860ab-2886-4e7c-96c8-11edf93fb680)